source/src/cip/cipconnectionmanager.c

```diff
@@ -1213,7 +1213,7 @@ EipUint8 ParseConnectionPath(
       }
 
       /* connection end point has to be the message router instance 1 */
-      if ( (class_id != kCipMessageRouterClassCode)
+      if ( (class_id != kCipMessageRouterClassId)
```

Comment:
same here, official name Class Code

Reply:
same revert here ..., but you see it here compared with "class_id"